documentation/sphinx/source/api-python.rst

```diff
@@ -591,14 +591,18 @@ Writing data
 
     Removes all keys ``k`` such that ``begin <= k < end``, and their associated values. |immediate-return|
 
+    |transaction-clear-range-blurb|
+
     .. note :: Unlike in the case of :meth:`get_range`, ``begin`` and ``end`` must be keys (byte strings), not :class:`KeySelector`\ s.  (Resolving arbitrary key selectors would prevent this method from returning immediately, introducing concurrency issues.)
 
 ``del tr[begin:end]``
     Shorthand for ``tr.clear_range(begin,end)``. |slice-defaults|
 
 .. method:: Transaction.clear_range_startswith(prefix)
 
-    Removes all the keys ``k`` such that ``k.startswith(prefix)``, and their associated values. |immediate-return|
+    Removes all keys ``k`` such that ``begin <= k < end``, and their associated values. |immediate-return|
```

Comment:
This line should to be reverted to match the behavior of `clear_range_startswith` rather than `clear_range`.

Reply:
oops.. thanks for catching this - reverted.